index.html

```diff
@@ -7,38 +6,289 @@
+                        <li><a href="#">KID’S &amp; BABY</a></li>
+                        <li><a href="#">BAGS &amp; PURSES</a></li>                                           
+                    </ul>
+                </li>
+                <li> <a href="#">Home &amp; Living</a>
+                    <ul class="submenu">
+                        <li><a href="#">HOME</a></li>
+                        <li><a href="#">BATH &amp; BEAUTY</a></li>
+                        <li><a href="#">PET SUPPLIES</a></li>                                        
+                    </ul>
+                </li> 
+                <li> <a href="#">Wedding &amp; Party</a>
+                <ul class="submenu">
+                        <li><a href="#">PARTY SUPPLIES</a></li>
+                        <li><a href="#">INVITATIONS &amp; PAPER</a></li>
+                        <li><a href="#">WEDDING DECORATIONS</a></li>
+                        <li><a href="#">WEDDING GIFTS</a></li>
+                        <li><a href="#">WEDDING ACCESSORIES</a></li>
+                        <li><a href="#">WEDDING CLOTHING</a></li>   
+                        <li><a href="#">WEDDING JEWERLY</a></li>                                             
+                    </ul>
+                </li>
+                <li> <a href="#">Toys &amp; Entertainment</a>
+                    <ul class="submenu">
+                        <li><a href="#">TOYS</a></li>
+                        <li><a href="#">ELECTRONICS &amp; ACCESSORIES</a></li>
+                        <li><a href="#">BOOKS</a></li>
+                        <li><a href="#">MOVIES &amp; MUSIC</a></li>                                           
+                    </ul>
+                </li> 
+                <li> <a href="#">Art &amp; Collectibles</a>
+                    <ul class="submenu">
+                        <li><a href="#">PRINTS</a></li>
+                        <li><a href="#">PHOTOGRAPHY</a></li>
+                        <li><a href="#">PAINTING</a></li>
+                        <li><a href="#">SCULPTURE</a></li>  
+                        <li><a href="#">COLLECTIBLES</a></li>                                           
+                    </ul>
+                </li>
+                <li> <a href="#">Craft Supplies &amp; Tools</a>
+                    <ul class="submenu">
+                        <li><a href="#">HOME &amp; HOBBY</a></li>
+                        <li><a href="#">SEWING &amp; FIBER</a></li>
+                        <li><a href="#">PAPERCRAFT</a></li>
+                        <li><a href="#">VISUAL ARTS</a></li>  
+                        <li><a href="#">SCULPTING &amp; FORMING </a></li>                                           
+                    </ul>
+                </li>
                 <li> <a href="#">Vintage</a></li>
-                
             </ul>
         </nav>
-
     </header>
+    <main>
+        <section class="firstSection">
+                <ul id= 'Slider' class="slider">
```

Comment:
id should not have space, and the value should be on quotes and not single quotes